diff --git a/node_modules/webpack-auto-inject-version/dist/WebpackAutoInjectVersion.js b/node_modules/webpack-auto-inject-version/dist/WebpackAutoInjectVersion.js
index 7f672ae..b6fca30 100644
--- a/node_modules/webpack-auto-inject-version/dist/WebpackAutoInjectVersion.js
+++ b/node_modules/webpack-auto-inject-version/dist/WebpackAutoInjectVersion.js
@@ -3066,7 +3066,7 @@ var AutoIncreaseVersion = function () {
       // we have to register AutoIncreaseVersion instead of firing it straight away
       if (_config2.default.componentsOptions.AutoIncreaseVersion.runInWatchMode) {
         if (this.context.compiler) {
-          this.context.compiler.plugin('emit', function (compilation, cb) {
+          this.context.compiler.hooks.emit.tapAsync('emit', function (compilation, cb) {
             _this.start();
             cb();
           });
@@ -3276,7 +3276,7 @@ var InjectAsComment = function () {
       var _this = this;

       // bind into emit hook
-      this.context.compiler.plugin('emit', function (compilation, cb) {
+      this.context.compiler.hooks.emit.tapAsync('emit', function (compilation, cb) {
         // iterate over all assets file in compilation
         for (var basename in compilation.assets) {
           // bug fix, extname is not able to handle chunk file params index.js?random123
@@ -3508,7 +3508,7 @@ var InjectByTag = function () {
     value: function apply() {
       var _this = this;

-      this.context.compiler.plugin('emit', function (compilation, cb) {
+      this.context.compiler.hooks.emit.tapAsync('emit', function (compilation, cb) {
         // for every output file
         for (var basename in compilation.assets) {
           // only if match regex
